service/web/tabs/app/src/containers/CreateFormContainer.tsx

```diff
@@ -291,6 +293,10 @@ class CreateFormContainer extends React.Component<
   private nullIfEmpty(str: string) {
     return str !== undefined && str !== null && str.length === 0 ? null : str
   }
+
+  private base64(str: string) {
+    return str ? btoa(str) : str
```

Comment:
btoa?

Reply:
welcome to javascript.. base64. Hilariously, `btoa(undefined) == btoa("undefined")`

Reply:
🤦